apps/web-giddh/src/app/ledger/components/advance-search/advance-search.component.scss

```diff
@@ -46,15 +46,30 @@
     .ledger-stat {
         margin: 0px 0 0;
     }
-    .ledger_book {
+    .ledger-book {
         margin-bottom: 100px;
     }
     .ledger-head thead tr th.active {
         display: block !important;
         width: 100%;
     }
-    .ledger_book .h-table tbody .ledger-row td {
+    .ledger-book .h-table tbody .ledger-row td {
         position: relative;
         padding-left: 80px !important;
     }
+    .Advance-Search{
```

Comment:
Do use correct naming convention for classes

Reply:
@shalinee-walkover Please change it everywhere where -ever it is used